packages/nodes-base/nodes/SendInBlue/SendInBlueTrigger.node.ts

```diff
@@ -0,0 +1,300 @@
+/* eslint-disable n8n-nodes-base/node-filename-against-convention */
+import {
+	IDataObject,
+	IHookFunctions,
+	INodeType,
+	INodeTypeDescription,
+	IWebhookFunctions,
+	IWebhookResponseData,
+	NodeOperationError,
+} from 'n8n-workflow';
+import { SendInBlueWebhookApi } from './GenericFunctions';
+
+export class SendInBlueTrigger implements INodeType {
+	description: INodeTypeDescription = {
+		credentials: [
+			{
+				name: 'sendinblueApi',
+				required: true,
+			},
+		],
+		displayName: 'SendInBlue Trigger',
+		defaults: {
+			name: 'SendInBlue-Trigger',
+			color: '#044a75',
+		},
+		description: 'Starts the workflow when SendInBlue events occur',
+		group: ['trigger'],
+		icon: 'file:sendinblue.svg',
+		inputs: [],
+		name: 'sendinblueTrigger',
+		outputs: ['main'],
+		version: 1,
+		webhooks: [
+			{
+				name: 'default',
+				httpMethod: 'POST',
+				responseMode: 'onReceived',
+				path: 'webhooks',
+				responseData: 'allEntries',
```

Comment:
This option can probably be removed. As it is returns the data "onReceived" and there are so no entries to return.